tools/docker/devmapper-snapshotter/config.toml

```diff
@@ -0,0 +1,6 @@
+disabled_plugins = ["cri"]
```

Comment:
nit: I think we don't need this line, I'm not sure why we cary this everywhere.

Reply:
Thanks. Will remove.

Reply:
Removed the line from tools/docker/config.toml, which is shared by all integration tests.